src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlInternalConnectionTds.cs

```diff
@@ -3075,7 +3076,7 @@ internal void OnFeatureExtAck(int featureId, byte[] data)
                             throw SQL.ParsingError(ParsingErrorState.CorruptedTdsStream);
                         }
                         byte enabled = data[1];
-                        _parser.DataClassificationVersion = (enabled == 0) ? TdsEnums.DATA_CLASSIFICATION_NOT_ENABLED : supportedDataClassificationVersion;
+                        _parser.DataClassificationVersion = (enabled == 0) ? TdsEnums.DATA_CLASSIFICATION_NOT_ENABLED : _supportedDataClassificationVersion;
```

Comment:
Is this copy of the same information needed? The parser and internal connection are very solidly linked together so I think it might be possible to just keep one copy and refer to it using a getter property if needed.

While working with the current library I find that the copying of data around makes auditing usage of variables very time consuming and the existing habit of using direct access to internal members of other classes does the same.